common/src/main/scala/akka/contrib/persistence/mongodb/MongoReadJournal.scala

```diff
@@ -72,9 +73,12 @@ class ScalaDslMongoReadJournal(impl: MongoPersistenceReadJournallingApi)(implici
 
   def allEvents(offset: Long): Source[EventEnvelope, NotUsed] = {
     val pastSource = impl.currentAllEvents(offset)
-    val realtimeSource = Source.actorRef(100, OverflowStrategy.dropTail).mapMaterializedValue(impl.subscribeJournalEvents)
+    val realtimeSource = Source.actorRef[Event](100, OverflowStrategy.dropTail).mapMaterializedValue(ref => {
+                                                                                  impl.subscribeJournalEvents(ref)
+                                                                                  NotUsed
+                                                                                })
     (pastSource ++ realtimeSource)
-      .filter(_.timestamp >= offset)
+      .filter(_.timestamp > offset)
```

Comment:
I'm still not convinced this is the right way to go though. I understand the limitations of the timestamp but:

1. this goes against the spec, the docs clearly says it's exclusive: http://doc.akka.io/docs/akka/current/scala/persistence-query.html
2. the `RemoveDuplicatedEventsByPersistenceId` always starts as an empty HashMap, so when it restarts it does not have any knowledge about the last persistence IDs cause they aren't loaded from anywhere, this would require users of the library to know that they have to store the map themselves and apply additional filters.